### PR TITLE
Remove Link Command from UG, fix typos

### LINKED ISSUE #134: Remove Link command from UG
Link command removed since #125

### PR DESCRIPTION
**Pull request by marclamp**
Resolves #134

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -104,12 +103,14 @@ Format: `help`
 
 Adds a student to the address book.
 
-Format: `add n/NAME s/STUDENT_ID e/EMAIL h/TELEGRAM_HANDLE [c/TIMETABLE] [t/TAG]…​`
+Format: `add n/NAME s/STUDENT_ID e/EMAIL h/TELEGRAM_HANDLE [l/WEBLINK] [c/TIMETABLE] [t/TAG]…​`
```
Comment (jeremiahlzz) [inline docs/UserGuide.md:106]: Thanks for updating the add command syntax

**Review by jeremiahlzz — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -71,7 +70,7 @@ EduConnect is a **desktop app for managing student contacts, optimized for use v
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g. `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```
Comment (fy17ohhh) [inline docs/UserGuide.md:73]: thx for finding this small typo

**Review by fy17ohhh — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -133,7 +134,7 @@ Format: `edit INDEX [n/NAME] [s/STUDENT_ID] [e/EMAIL] [h/TELEGRAM_HANDLE] [l/WEB
 * Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the student will be removed i.e adding of tags is not cumulative.
+* When editing tags, the existing tags of the student will be removed i.e. adding of tags is not cumulative.
```
Comment (tanjiajiajun) [inline docs/UserGuide.md:137]: Good job making the usage of i.e consistent :)

**Review by tanjiajiajun — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (jeremiahlzz) [reply docs/UserGuide.md:106]: part of #119

**Pull request merged**